contracts/HintHelpers.sol

```diff
@@ -19,6 +20,7 @@ contract HintHelpers is VestaBase, CheckContract {
 
 	ISortedTroves public sortedTroves;
 	ITroveManager public troveManager;
+	IPriceFeed public priceFeed;
```

Comment:
Updated on my end, but this is breaking the storage. You are collapsing with the old storage structure. When adding new variable in an upgradeable scenario it should be under all existing variables.